.ci/code/Versioning_Test/Verify/FromJson.cs

```diff
@@ -44,10 +44,13 @@ public static partial class Verify
         /**** Test Methods                ****/
         /*************************************/
 
-        public static TestResult FromJsonDatasets()
+        public static TestResult FromJsonDatasets(bool testAll = false)
```

Comment:
Wonder if this should not be defaulting to true, to keep the current behaviour, unless instructed otherwise by the caller. Will still approve and leave it up to you if you want to change this or not.

```suggestion
        public static TestResult FromJsonDatasets(bool testAll = true)
```